### PR TITLE
Add support for customizing profile and analyze action

### PR DESCRIPTION
**Pull request by zhuhaow**
Resolves https://github.com/tuist/tuist/issues/893

### CODE REVIEW

File: Sources/ProjectDescription/AnalyzeAction.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct AnalyzeAction: Equatable, Codable {
```
Comment (ollieatkinson) [inline Sources/ProjectDescription/AnalyzeAction.swift]:
```suggestion
public struct AnalyseAction: Equatable, Codable {
```
This should be Analyse to follow with Xcode's naming convention

Comment (vytis) [reply Sources/ProjectDescription/AnalyzeAction.swift]: I think it's done properly and localized as it should :) Screenshot 2020-02-07 at 3 44 54 PM
As a point of reference at least in frameworks it's mostly US spelling e.g. `UIColor.gray`.

Comment (ollieatkinson) [reply Sources/ProjectDescription/AnalyzeAction.swift]: 👍🏼

**Comment by zhuhaow** [comment]
Hi @pepibumur , sorry for the late reply.

I added some test in fixtures/ios_app_with_multi_configs/App/Project.swift and it worked fine.

**Comment by pepibumur** [comment]
@zhuhaow would you mind solving the git conflicts? We can merge afterward

**Pull request merged**